README.md

```diff
@@ -1,199 +1,199 @@
 
 > [!TIP]
-> Update Jan 10, 2025: **Applications for cohort 2 will open soon.** Please check back here for updates.
+> Update Jan 21, 2025: **Applications for cohort 2 will open soon.** Please check back here for updates.
 
 
-# ink!ubator: Ecosystem Grants<!-- omit in toc -->
+# ink!ubator 2.0 <!-- omit in toc -->
 
-- [:wave: Introduction](#wave-introduction)
-  - [Guidelines](#guidelines)
-  - [Project Ideas](#project-ideas)
-  - [Bounty Levels](#bounty-levels)
-  - [Support](#support)
-  - [Team](#team)
-- [:pencil: Process](#pencil-process)
+- [\:wave: Introduction](#wave-introduction)
+- [Guidelines](#guidelines)
+- [Project Ideas](#project-ideas)
+- [Bounty Levels](#bounty-levels)
+- [Support](#support)
+- [Team](#team)
+- [\:pencil: Process](#pencil-process)
   - [1. Application](#1-application)
   - [2. Application Review](#2-application-review)
   - [3. Milestone Delivery and Payment](#3-milestone-delivery-and-payment)
   - [Changes to a Grant after Approval](#changes-to-a-grant-after-approval)
-- [:mailbox\_with\_mail: Suggest a Project](#mailbox_with_mail-suggest-a-project)
-- [:bulb: Help](#bulb-help)
+- [\:mailbox_with_mail: Suggest a Project](#mailbox_with_mail-suggest-a-project)
+- [\:bulb: Help](#bulb-help)
   - [Real-time conversation](#real-time-conversation)
 - [FAQ](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/FAQ.md)
 
 <img src="./assets/inkubator-tanks.svg" style="width: 100%;">
 
-## :wave: Introduction
+**👋 Introduction**
+
 Ecosystem Grants are part of the **ink!ubator**, a holistic bounty program with the goal of furthering the ink! smart contracting ecosystem. This bounty program is funded by the Polkadot treasury with the support of the community.
 
 The bounty includes Curators from the following Polkadot ecosystem teams:
-- [Astar Network](https://astar.network/)
-- [Parity](https://www.parity.io/)
-- [Phala Network](https://www.phala.network/en/)
-- [Brushfam](https://brushfam.io/)
-- [Aleph Zero](https://alephzero.org/)
 
-The Ecosystem Grants focus on funding projects which will demonstrate the potential of ink! and well as those which push the ink! ecosystem forward, such as infrastructure projects or developer tooling.
+- [W3F](https://web3.foundation/)
+- [Michi](https://github.com/cmichi)
+- [Velocity Labs](https://www.velocitylabs.org/)
+- [David Semakula](https://davidsemakula.com/)
+- [R0GUE](https://github.com/al3mart)
+- **Facilitator**:
+    - [Toma Sadova](https://www.linkedin.com/in/toma-sadova/)
+
+The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative, see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).
+
+---
 
-The Ecosystem Grants are only one aspect of the ink!ubator. For more details on the rest of the initiative see the [Polkadot Treasury Bounty Proposal](https://polkadot.polkassembly.io/bounty/19).
+**Guidelines**
 
-### Guidelines
```

Comment:
If you remove the headings ("\#\#\#"), the links in the table of contents won't work anymore.